docs/operator-guides/scaling-airbyte.md

```diff
@@ -66,3 +66,42 @@ The advice here is best-effort and by no means comprehensive. Please reach out o
 
 If you've been running Airbyte in production and have more tips up your sleeve, we welcome contributions!
 
+## Recommended Metrics
+Airbyte supports exporting built-in metrics to Datadog or [OpenTelemetry](https://docs.airbyte.com/operator-guides/collecting-metrics/)
+
+### Key Metrics
+* `oldest_pending_job_age_secs`
+    * This metric shows how long a pending job waits before it is scheduled. If a job is in pending state for a long time, more workers may be required.
```

Comment:
After reformatting this section as a table, you can delete the phrase "this metric" and start the descriptions with the verb ("shows...")

Reply:
Done :)